electron-app/src/views/Dashboard.vue

```diff
@@ -1,46 +1,75 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col>
-        <h1 class="page-header">Dashboard</h1>
+    <v-row style="margin: 0;">
+      <v-col cols="12">
+        <overall-balances></overall-balances>
       </v-col>
     </v-row>
-
-    <div class="dashboard__information-boxes">
-      <exchange-box
-        v-for="exchange in exchanges"
-        :key="exchange.name"
-        :name="exchange.name"
-        :amount="exchange.total"
-      ></exchange-box>
-      <information-box
-        v-if="!blockchainTotal.isZero()"
-        id="blockchain_box"
-        icon="fa-hdd-o"
-        :amount="blockchainTotal"
-        blockchain
-      ></information-box>
-      <information-box
-        v-if="!fiatTotal.isZero()"
-        id="banks_box"
-        icon="fa-university"
-        :amount="fiatTotal"
-      ></information-box>
-    </div>
-
-    <v-row>
-      <v-col cols="12">
-        <h1 class="page-header">All Balances</h1>
+    <v-row class="flex" style="margin: 0; justify-content: space-evenly;">
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="exchange"
+          can-refresh
+          :is-loading="exchangeIsLoading"
+          @refresh="refreshBalance($event)"
+        >
+          <exchange-box-new
+            v-for="exchange in exchanges"
+            :key="exchange.name + '_new'"
+            :name="exchange.name"
+            :amount="exchange.total"
+          ></exchange-box-new>
+        </summary-card>
+      </v-col>
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="blockchain"
+          :is-loading="blockchainIsLoading"
+          can-refresh
+          @refresh="refreshBalance($event)"
+        >
+          <blockchain-balance-card-list
+            v-for="(usdValue, protocol) in blockchainTotals"
+            :key="protocol"
+            :name="protocol"
+            :amount="usdValue"
+          ></blockchain-balance-card-list>
+        </summary-card>
+      </v-col>
+      <v-col cols="12" md="4" lg="4">
+        <summary-card name="manual">
+          <manual-balance-card-list
```

Comment:
Same idea about sending the user to the manually tracked balances view when he clicks here.

Reply:
Would be great if it could actually send to manually tracked balances (pre)filtered by that location...

Reply:
Is that doable or not for this PR?

Reply:
Not for this one, no.